### DISCUSSION TITLE
Twitter Profile review live stream retrospective

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#4580] [General]
LIVE stream with amazing guests reviewing and shouting out YOUR Twitter profiles https://www.youtube.com/watch?v=InNQPc_7gps

Please share what you liked and didn't like below, so we can improve for next time

**Comment by eddiejaoude** [comment]
My thoughts...

- enable slow mode
- close issues after review lol
- maybe limit how long we spend on each review, so we can do more

**Comment by naomi-lgbt** [comment]
I think we should approach this like we do with our open source reviews:

Start from the oldest issue and work our way through the list.

**Comment by ewainy** [comment]
I think a mix of the oldest issues and then some people that are joining in the chat 😊

If people spam their issue number/twitter handle then they don't get picked or is that a bit harsh idk lol 🤷‍♀️

I like the idea to find out what their goal is for their account so we can give tailored advice. Like YK was saying.

**Reply by naomi-lgbt** [reply]
Coming from the moderation standpoint, handling the issue numbers in chat was an absolute nightmare.

Things were going so fast that I was struggling to keep up with the timeouts.

This might be addressed by slowmode, but the challenge will be finding a cooldown time that prevents the spam but doesn't stifle conversation.

**Reply by ewainy** [reply]
I've never seen anything like it 🤯

Doing whatever is possible to minimise that happening again definitely needs to be the focus and perhaps it can be explained that the approach will be x because of this reason etc 🙂

**Comment by 44jax44** [comment]
I think it shouldn't be a competition in the chat because, at the start of the stream we were commenting relevant things to what they were saying, it was fun, we can join in on the reviews and add our points and thoughts and the chat would be directed that way.

Then it became people spamming numbers and we couldn't basically "discuss" the stream and join in on the fun because everybody was just spamming 

The chat became very annoying, disrespectful and closing it was the best option for me at least 

- I think the names should be picked beforehand. Wanna do old to new that's great, wanna pick at random that also works
- Before starting the stream have an agenda, we can do 15 profiles let's say, pick 30 profiles and go through them
- Call out the person in the chat let them respond make it known that they are there and then do the rating 
- If they are not there you have 15 backup profiles you picked 

That way you have your profiles, you don't need to announce it so people don't know who's gonna be up, and most importantly the chat would be only directed to discuss the profiles and not spam, no slow mode would be needed there but a couple of seconds slow mode wouldn't hurt I guess

**Reply by ewainy** [reply]
Yes I thought that part was a real shame, I've been on many lives where
Eddie reviews things and the chat is usually very loving and supportive and
people add their views and we all help eachother. I agree it was like a
competition this time and things can be improved for next time with getting
feedback ❤️
